client/src/js/directives/findDebtorGroup.js

```diff
@@ -0,0 +1,116 @@
+/**
+* <bh-find-debtorgroup></bh-find-debtorgroup>
+*
+* The find debtor group component is responsible for displaying a component
+* for searching a debtor group or a convention
+*
+* This component takes these attributes :
+*   - locked : if we want only debtor groups which are locked or not
+*   - is-convention : if we want only debtor groups which are conventions or not
+*   - on-search-complete : the callback function which get the returned debtror group
+*
+* @example
+* // component find debtor group
+* <bh-find-debtorgroup>
+*   locked="true"
+*   is-convention="true"
+*   on-search-complete="MyCtrl.handleResult"
+* </bh-find-debtorgroup>
+*/
+
+/** Find Debtor Group Controller */
+function findDebtorGroupController($http, DebtorGroupService) {
+  var ctrl         = this,
+      callback     = this.callback,
+      locked       = this.locked,
+      isConvention = this.isConvention;
+
+  /** Load debtor group */
+  loadDebtorGroup();
+
+  /**
+  * loading state :
+  * This variable takes these values (0|1|2) :
+  *   - 0 = loading
+  *   - 1 = loaded
+  *   - 2 = error
```

Comment:
These are good comments.  Using integers for state is not a good idea for the future, but this is well commented, so thank you.  We will have to figure out a better way to do module state in the future.